README.md

```diff
@@ -198,6 +198,7 @@ metadata:
   name: "test"
   annotations:
     dbtdoc-bucket: "my-bucket"
+    dbtdoc-bucket-path: "optional/override/path" # Optional
```

Comment:
Maybe use the key `dbtdoc-path` instead of `dbtdoc-bucket-path`, update the code and readme when needed